specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/scenarios/virtualMachine.yaml

```diff
@@ -0,0 +1,40 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/documentation/api-scenario/references/v1.2/schema.json
+scope: ResourceGroup
+
+variables:
+  subnetID:
+    type: string
+  publicIPID:
+    type: string
+
+scenarios:
+  - scenario: VirtualMachine
+    description: Microsoft.Compute/virtualMachines CRUD
+    steps:
+      - step: NetworkAndSubnet_Create
+        armTemplate: peArmTemplate/Create_Network_And_Subnet.json
+      - step: PublicIpAddress_Create
+        armTemplate: peArmTemplate/Create_PublicIpAddress.json
+      - step: NetworkInterface_Create
+        armTemplate: peArmTemplate/Create_NetworkInterface.json
+      - step: VirtualMachine_Create
+        exampleFile: ../examples/compute/CreateAWindowsVmWithPatchSettingAssessmentModeOfImageDefault.json
+        requestUpdate:
+          - replace: /parameters/properties/networkProfile/networkInterfaces/0/id
+            value: $(nicID)
+          - replace: /parameters/properties/osProfile/adminUsername
+            value: goscenario
+          - replace: /parameters/properties/osProfile/adminPassword
+            value: QWE!@#123
```

Comment:
password is better to be `secureString` and pass in at runtime

Reply:
Replace adminPwd with type secureString